examples/mnist-classification/src/mnist_classification/core.clj

```diff
@@ -0,0 +1,214 @@
+(ns mnist-classification.core
+  (:require [clojure.java.io :as io]
+            [cortex.datasets.mnist :as mnist]
+            [mikera.image.core :as i]
+            [think.image.image :as image]
+            [think.image.patch :as patch]
+            [think.image.data-augmentation :as image-aug]
+            [cortex.nn.layers :as layers]
+            [clojure.core.matrix.macros :refer [c-for]]
+            [clojure.core.matrix :as m]
+            [cortex.experiment.classification :as classification]
+            [cortex.experiment.train :as train]
+            [cortex.nn.network :as network]
+            [cortex.nn.execute :as execute]
+            [cortex.util :as util]
+            [cortex.experiment.util :as experiment-util])
+  (:import [java.io File]))
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Convolutional neural net description
+(def image-size 28)
+(def num-classes 10)
+
+(def initial-description
```

Comment:
Better to make this a function that takes some default parameters (like perhaps the input size) which makes it more generic for people copying this for other classification situations.